marketplaceItems/spectatorCamera/spectatorCamera.js

```diff
@@ -18,6 +18,8 @@
     var sendToQml, addOrRemoveButton, onTabletScreenChanged, fromQml,
         onTabletButtonClicked, wireEventBridge, startup, shutdown, registerButtonMappings;
 
+    var vFoV = Settings.getValue("spectatorCamera/vFoV", 45.0);

```

Comment:
Please move this to go right above `spectatorCameraOn()` where it's first used.

Reply:
this was an old variable from when i thought i needed to fix the fov on the spectator camera. ive deleted it